tests/pass/shims/fs.rs

```diff
@@ -123,7 +120,9 @@ fn test_file_create_new() {
     remove_file(&path).unwrap();
 }
 
+#[cfg(not(all(target_os = "freebsd", target_pointer_width = "32")))]
```

Comment:
There's still something wrong here, why is this disabled on 32bit?
I said it multiple times: please don't just disable tests when they fail! Sometimes it is okay to cfg-out a particular test on a particular target, but that *always* needs a comment explaining why it is done. And here I would expect the test should pass on freebsd so when it fails that needs investigation, not `cfg`.

Reply:
I forgot to reenable this one (especially there is no cfg upon test_seek above), no need to assume bad intentions ..